apps/site/lib/site_web/views/page_content_view.ex

```diff
@@ -118,13 +122,16 @@ defmodule SiteWeb.CMS.PageView do
 
       alert_path ->
         alert_project_paths = get_project_url_paths(alert_path)
+        alert_project_paths = Enum.map(alert_project_paths, &trim_and_downcase/1)
```

Comment:
these 2 lines and the next I believe can be combined w/ the pipe operator like...

```elixir
alert_path
|> get_project_url_paths()
|> Enum.map(&trim_and_downcase/1)
|> Enum.any?(&MapSet.member?(project_paths, &1))
```